extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmConfig.java

```diff
@@ -279,10 +279,19 @@ public boolean isAnyPropertySet() {
         /**
          * Show SQL logs and format them nicely.
          * <p>
+         * true logs sql statements
+         * extended logs sql statements and bind parameter values
+         * <p>
          * Setting it to true is obviously not recommended in production.
          */
-        @ConfigItem(defaultValue = "false")
-        public boolean sql;
+        @ConfigItem(defaultValueDocumentation = "false")
```

Comment:
Shouldn't this be `defaultValue=false`?

Reply:
`defaultValue=false` would also work here, but would require a more in depth check in the `isAnyPropertySet()` method (line 303)

Reply:
I wouldn't do it this way. It would be better to create a separate parameter for this than to have properties which are boolean plus some other thing.